pallets/subtensor/tests/staking.rs

```diff
@@ -847,7 +847,7 @@ fn test_remove_balance_from_coldkey_account_ok() {
         // Should be able to withdraw without hassle
         let result =
             SubtensorModule::remove_balance_from_coldkey_account(&coldkey_account_id, ammount);
-        assert_eq!(result, true);
+        assert!(result.is_ok());
```

Comment:
why shake your gift when you can open it and see if it's broken directly ;)

```suggestion
        result.unwrap();
```